R/routes.r

```diff
@@ -1,5 +1,5 @@
 #' Determine the correct route to use based on a list of routes and
-#' the current path.
```

Comment:
Apparently path is a special word somewhere deep in R, and renaming the variable fixed my issues